### PR TITLE
Add donate page

### PR DESCRIPTION
**Pull request by davidak**
This adds a new donate page with information from community page.

I also improved the wording (i think). Feel free to suggest better wording.

Fixed layout of logos.

before:
Screenshot from 2020-04-18 18-31-18

after:
Screenshot from 2020-04-18 18-39-10

but with less than 755 pixel width, the logos look broken.
Screenshot from 2020-04-18 18-30-38

it's a dirty hack because the old bootstrap don't support [align-middle](https://getbootstrap.com/docs/4.4/utilities/vertical-align/)

### CODE REVIEW

File: layout.tt [hunk 1]
```diff
@@ -91,13 +93,15 @@
...
                 <li><a href="[%root%]governance.html">Governance</a></li>
+                <li><a href="[%root%]donate.html">Donate</a></li>
               </ul>
               <ul class="nav pull-right">
                 <li><a href="https://github.com/NixOS/nixpkgs"><i class="fa fa-github"></i></a></li>
                 [% ELSIF menu == 'organization' %]
                 <ul class="nav pull-right">
                   <li><a href="[%root%]learn.html">Learn</a></li>
                   <li><a href="[%root%]governance.html">Governance</a></li>
+                  <li><a href="[%root%]donate.html">Donate</a></li>
```
Comment (davidak) [inline layout.tt:104]: i don't know what site that is, but i added it also next to Learn and Governance

File: donate.tt [hunk 2]
```diff
@@ -0,0 +1,97 @@
+[% WRAPPER layout.tt hideTitle=1 title="Donate" menu='nixos' %]
+
+<h1>Donate</h1>
+
+<p class="lead">
+The NixOS infrastructure, development and events
+are funded by it's users and sponsors.
```
Comment (worldofpeace) [inline donate.tt:7]:
```suggestion
are funded by its users and sponsors.
```

**Comment by samueldr** [comment]
I've been working on re-doing the CSS to give us the ability to more simply work with it, instead of being stuck in bootstrap 2.

I've reworked the acknowledgments (logo) section's CSS so it works right on all browser widths, I'll soon open a PR with the changes, though I've been working on it here:

 * https://github.com/samueldr/nixos-homepage/tree/feature/re-css

So don't worry too much about the layout for existing issues. We'll see which PR gets in master first, but fixing one or the other should be trivial.

(I don't have a PR open yet because of the issue with creating previews on non-repo branches, and I often push and rebase)

**Comment by garbas** [comment]
Just merged community page and merged current master into this PR to remove the conflicts.
 
@samueldr Personally I would rather ignore the CSS and focus on getting towards a bigger redesign, but I understand that until then we will also have to move things around. For me bootstrap 2 is good for now. I especially want to avoid doing any style changes for now, since we will make decisions not looking at the big picture.

**Comment by davidak** [comment]
>Let me know what you think, for me it looks ready to be merged.

i agree. your changes are a huge improvement on the structure

**Comment by davidak** [comment]
>Let me know what you think, for me it looks ready to be merged.

i agree. your changes are a huge improvement on the structure

**Pull request merged**